package.json

```diff
@@ -68,10 +68,10 @@
     "eslint-plugin-react-hooks": "2.3.0",
     "eslint-plugin-sonarjs": "0.5.0",
     "jest": "24.1.0",
-    "prettier": "1.18.2",
+    "prettier": "1.19.1",
     "ts-jest": "24.0.0",
     "ts-node": "~7.0.0",
-    "tslint": "~5.11.0",
+    "tslint": "6.0.0",
```

Comment:
Surprised that they are still releasing version. Apparently this one is marked as deprecated on NPM as well. :) https://github.com/palantir/tslint/releases

Reply:
yeah, hopefully, we won't need it sooner than later :)

Reply:
stil lot's of angular projects are using it - the migration in that ecosystem is much slower (as various tools and angular template checkers have been build on tslint...)